### DISCUSSION TITLE
Can't login to my password store

### DISCUSSION THREAD

**Discussion by lalkakonus** [zeapo/Android-Password-Store#1281] [Q&A] [answered]
I have a repository at github with password store. Have troubles with configuring Password Store git server.
For example have a repository with url https://github.com/username/repository and two step auth enabled.
I enter credential in Settings -> Repository URL like : git@github.com:username/repository and chose password auth mode.
When app request to enter password i insert generated personal access tokens from guithub, press OK and get error message: "An error occurred during a Git operation. Exhausted available auth methods"

**Comment by msfjarvis** [comment]
~~Sorry for the vague error message, we're still working on improving them. The error you're seeing means that your password was incorrect, and the app couldn't find any other way to authenticate. Can you double check the personal access token?~~ Check Fabian's response below, I missed that the URL was actually for SSH not HTTPS.

**Comment by fmeum** [comment, marked as answer]
I think that personal access tokens ate only accepted via HTTPS. You also need to include the username in the URL like this:
```
https://username@github.com/username/repo`
```
If it's not your repository, the first and second username may differ.

**Reply by lalkakonus** [reply]
Yes, this works fine, thanks a lot!
